modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -278,22 +278,25 @@ export class Keychains implements IKeychains {
    * @return {Promise<KeychainsTriplet>} newly created User, Backup, and BitGo keys
    */
   async createMpc(params: CreateMpcOptions): Promise<KeychainsTriplet> {
-    if (_.isUndefined(params.passphrase)) {
-      throw new Error('missing required param passphrase');
-    }
-
     let MpcUtils;
     switch (params.multisigType) {
       case 'tss':
         MpcUtils = this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? ECDSAUtils.EcdsaUtils : EDDSAUtils.default;
         break;
       case 'blsdkg':
+        if (_.isUndefined(params.passphrase)) {
+          throw new Error('missing required param passphrase');
+        }
         MpcUtils = BlsUtils;
         break;
       default:
         throw new Error('Unsupported multi-sig type');
     }
 
+    if (_.isUndefined(params.passphrase)) {
```

Comment:
Given that you still have the check for the passphrase here, doesn't this mean this function still works the same?

Reply:
Yeah I saw Michael's comment and just thought it was a return not a break, will fix that.